### DISCUSSION TITLE
Using loaddll in scheduler jobs

### DISCUSSION THREAD

**Discussion by Umbrella-TEC** [Maplespe/DWMBlurGlass#302] [General]
In version 2.2 loaddll was removed. The EXE process is launched instead
I suggest keeping loaddll. You can monitor application crashes using Dwminit events. Is this possible? Example of a task with tracking:
[Aeroglass check.zip](https://github.com/Maplespe/DWMBlurGlass/files/15185891/Aeroglass.check.zip)

**Comment by Maplespe** [comment]
In fact, glass8 also uses an aerohost.exe background process.

DWMBlurGlass does this for more than just checking for crashes and reloading it, it also has multi-dwm instance support for multi-user (not fully completed) and RDP, and monitors each dwm process.

In addition, the background process of DWMBlurGlass takes up less than 5MB of memory and less than 0.5% of cpu, which does not cause any performance impact.

At the same time, due to dwm's permission limitations, it is not possible to read some configurations of the user (HKCU), so the DWMBlurGlass.exe process is resident, and is used by DWMBlurGlassExt to query settings from the main process, and to return Ext.
For example, query power, transparent status, system settings.
(The specific code is in "`DWMBlurGlassExt/Helper/Notifier.cpp`".)

Since DWMBlurGlass now features automatic injection, there is no reason for this option（`loaddll`） to remain.
